crates/runc-shim/src/task.rs

```diff
@@ -178,6 +189,11 @@ where
         Ok(Empty::new())
     }
 
+    fn close_io(&self, _ctx: &TtrpcContext, _req: CloseIORequest) -> TtrpcResult<Empty> {
+        // unnecessary close io here since fd was closed automatically after object was destroyed.
+        Ok(Empty::new())
```

Comment:
`FileDesc` inside the `File` has implemented `Drop` trait which will close fd after object was destroyed. As a result, I think `close_io` task is unnecessary.